src/helm.hoverProvider.ts

```diff
@@ -40,6 +42,15 @@ export class HelmTemplateHoverProvider implements vscode.HoverProvider {
         }
 
         if (this.notInAction(doc, pos, word)) {
+            try {
+                // have a test against whether the word is a key element in yaml
```

Comment:
Can we document why we care about this, rather than just what check we're performing?